### PR TITLE
US 35 & 49

### PR DESCRIPTION
**Pull request by dborski**
Adds all pending orders for each merchant to merchant dashboard

Adds following info to merchant/orders show page:
* Recipient's name and address
* Name of each item as link to item's show page
* Image of each item
* Price of each item
* The quantity the user wants to purchase

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Order <ApplicationRecord
   belongs_to :user
   has_many :item_orders
   has_many :items, through: :item_orders
+  enum status: {pending: 0, packaged: 1, shipped: 2, cancelled: 3}
```
Comment (dborski) [inline app/models/order.rb:7]: I just moved this above model methods.

File: app/models/user.rb [hunk 2]
```diff
@@ -13,4 +13,8 @@ class User < ApplicationRecord
   has_secure_password
 
   enum role: {regular: 0, merchant: 1, admin: 2}
+
+  def full_address
+    "#{address}, #{city}, #{state}, #{zip}"
+  end
```
Comment (dborski) [inline app/models/user.rb:19]: Address helper method. Could probably add this to all models with an address.

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -160,3 +160,8 @@ tr:nth-child(even) {
  .white {
  color: white;
  }
+
+ img.resize {
```
Comment (perryr16) [inline app/assets/stylesheets/application.css:164]: style king

File: app/models/order.rb [hunk 4]
```diff
@@ -13,5 +14,9 @@ def total_items
     items.sum(:quantity)
   end
 
-  enum status: {pending: 0, packaged: 1, shipped: 2, cancelled: 3}
+  def self.pending_merchant_orders(merchant_id)
+    Order.joins(:item_orders => :item)
```
Comment (perryr16) [inline app/models/order.rb:18]: nice AR call

**Comment by foleymichelle9** [comment]
Looks good! I approve merging

**Pull request merged**